backend/utils/sellers.js

```diff
@@ -14,20 +15,23 @@ async function createSeller({ name, email, password, superuser }) {
     return { status: 409, error: 'Registration exists' }
   }
 
-  const numSellers = await Seller.count()
   const salt = await createSalt()
   const passwordHash = await hashPassword(salt, password)
 
   const seller = await Seller.create({
     name,
     email,
     password: passwordHash,
-    superuser: superuser || (numSellers === 0 ? true : false) // First seller is superUser
+    superuser: superuser
   })
 
   return { seller }
 }
 
+async function numSellers() {
+  return await Seller.count()
+}
+
 async function findSeller(email) {
   const seller = await Seller.findOne({ where: { email } })
```

Comment:
Document that the email arg should be lower case or have this method lower case when doing the query.

Reply:
Lowercased.